src/vsr/replica.zig

```diff
@@ -580,12 +589,7 @@ pub fn ReplicaType(
                 }
             }
 
-            assert(
-                (self.status == .recovering and self.solo()) or
-                    self.status == .normal or
-                    self.status == .view_change or
-                    self.status == .recovering_head,
-            );
```

Comment:
My original thinking here was to signify that we can be at any status after `open`. So perhaps we could add 

```
maybe(self.status == .normal);
maybe(self.status == .view_change);
maybe(self.status == .recovering_head);
```